modules/client-configuration/pages/manual-registration/salt-registration.adoc

```diff
@@ -110,3 +110,28 @@ You can adjust this manually to correct the error and register the cloned system
 
 
 For more information and instructions, see pass:c[xref:FILENAME.adoc#bp.chapt.suma3.troubleshooting.registering.cloned.salt.systems[]].
+
+
+=== SSL errors
+
+On {slsa}{nbsp}11 systems, salt clients can sometimes have SSL errors which make some operations unusable, including package management and bootstrapping.
```

Comment:
This is not only happening with salt minions, traditional clients are affected as well and even more as the registration already fail with this error.

As salt itself does not use any SSL i think it would be better to just say any SLES11 client.

Reply:
In that case, I have this in the wrong section.

Reply:
Moved all the troubleshooting bits into their own section.